activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb

```diff
@@ -1,6 +1,7 @@
 require 'active_record/connection_adapters/abstract_mysql_adapter'
+require 'active_record/connection_adapters/mysql/database_statements'
 
-gem 'mysql2', '>= 0.3.18', '< 0.5'
+gem 'mysql2', '~> 0.4.4'
```

Comment:
Unfortunately, mysql2 0.4.3 was broken without prepared statements.
I would like to skip this version.